hack/go-test.sh

```diff
@@ -13,23 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-coverage=$1
 
-. ./hack/go-mod-env.sh
-
-if [[ -z ${CI} ]]; then
-    ./hack/go-fmt.sh
-    ./hack/go-lint.sh
-    ./hack/addheaders.sh
-fi
-setGoModEnv
-# using p flag to not run cmd tests in parallel, causing problems during config file read
-if [[ "${coverage}" == "true" ]]; then
-  go test -coverprofile cp_cmd.out ./cmd/... -count=1 -p=1
-  go test -coverprofile cp_pkg.out ./pkg/... -count=1
-  go tool cover -html=cp_cmd.out
-  go tool cover -html=cp_pkg.out
-else
-  go test ./cmd/... -count=1 -p=1
-  go test ./pkg/... -count=1
+if [[ -z ${ENVTEST_ASSETS_DIR} ]]; then
+  ENVTEST_ASSETS_DIR=testbin
 fi
+
+mkdir -p $(pwd)/${ENVTEST_ASSETS_DIR}
```

Comment:
maybe store the result of `$(pwd)/${ENVTEST_ASSETS_DIR}` in a var to be reused after instead of repeating the command each time

Reply:
Changed thanks :)